api/src/database/index.ts

```diff
@@ -127,3 +129,17 @@ export async function isInstalled(): Promise<boolean> {
 	// exists when using the installer CLI.
 	return await inspector.hasTable('directus_collections');
 }
+
+export const validateMigrations = async (): Promise<boolean> => {
+	const database = getDatabase();
+	try {
+		const completedMigrations = await database.select<Migration[]>('*').from('directus_migrations').orderBy('version');
+		const completedMigrationFiles = completedMigrations.map((migration) => {
+			return `${migration.version}-${migration.name.toLowerCase()}.ts`.replace(/ /g, '-');
```

Comment:
This will likely break in production as `.ts` files are compiled to `.js` files.